ariadne/cardano/test/backend/Test/Spec/Fixture.hs

```diff
@@ -58,14 +76,18 @@ withPassiveWalletFixture :: MonadIO m
 withPassiveWalletFixture pm prepareFixtures cc = do
     generateFixtures <- prepareFixtures
     liftIO $ Keystore.bracketTestKeystore $ \keystore -> do
-        bracketKernelPassiveWallet pm devNull keystore $ \layer wallet -> do
+        bracketKernelPassiveWallet pm devNull keystore Memory $ \layer wallet -> do
             fixtures <- generateFixtures wallet
             cc keystore layer wallet fixtures
 
 inMemoryDBComponent
     :: ComponentM (AcidState Kernel.DB)
 inMemoryDBComponent = buildComponent_ "In-memory DB" (openMemoryState defDB)
 
+tempDBComponent
+    :: FilePath -> ComponentM (AcidState Kernel.DB)
+tempDBComponent path = buildComponent_ "Temp Storage DB" (openLocalStateFrom path defDB)
```

Comment:
It's not really essential in tests, but I think a good style would be to use `closeAcidState` here.

Reply:
Also consider defining `walletDBComponent :: FilePath -> ComponentM (AcidState Kernel.DB)` in `Ariadne.Wallet.Cardano.WalletLayer.Kernel`, using it in `passiveWalletLayerComponent` and exporting solely for tests.

Reply:
Ok. done.